LayoutTests/inspector/runtime/resources/property-descriptor-utilities.js

```diff
@@ -27,6 +51,30 @@ class PrivateMembersTestClassParent {
 }
 
 class PrivateMembersTestClassChild extends PrivateMembersTestClassParent {
+    instancePublicProperty = 'instancePublicPropertyValue child';
+    #instancePrivateProperty = 'instancePrivatePropertyValue child';
+    instancePublicMethod() { child }
+    #instancePrivateMethod() { child }
+    get instancePublicGetter() { child }
+    set instancePublicSetter(x) { child }
+    get instancePublicGetterSetter() { child }
+    set instancePublicGetterSetter(x) { child }
+    get #instancePrivateGetter() { child }
+    set #instancePrivateSetter(x) { child }
+    get #instancePrivateGetterSetter() { child }
+    set #instancePrivateGetterSetter(x) { child }
+    static classPublicProperty = 'classPublicPropertyValue child';
+    static #classPrivateProperty = 'classPrivatePropertyValue child';
```

Comment:
Should this appear somewhere in your output? I don't see it if so.

Reply:
[LayoutTests/inspector/runtime/getDisplayableProperties-expected.txt:152](https://github.com/WebKit/WebKit/pull/13448/files#diff-a4beb82f7378e7e9b9b285825d88387629c0b16f05ca36fd2a5580f2c70e3543R152)
[LayoutTests/inspector/runtime/getProperties-expected.txt:134](https://github.com/WebKit/WebKit/pull/13448/files#diff-f594c930ddabcad1999b2b74a9f82be7773b1a9c65f1035cfb88fec84882932eR134)

also the `'classPrivatePropertyValue parent'` would be visible if we tested `PrivateMembersTestClassParent`